scripts/base/init-bare.zeek

```diff
@@ -5496,4 +5496,8 @@ event net_done(t: time)
 @if ( __init_primary_bifs() )
 @endif
 
+## A hook taking a packet timestamp. Called when a packet reaches the end of
+## processing without having been marked processed.
+global packet_not_processed: hook(pkt: pcap_packet);
```

Comment:
Now that I'm thinking about this, in script-land this should probably be an event, not a hook; similar to `new_packet`. At some level, I am still wondering if we should offer this at all. I'm fine keeping it, but woudkl suggest to add a note to the comment that handling this could be expensive if there's lots of unprocessed. traffic

Reply:
I think removing it is fine. We don't explicitly do this for the other types of plugin hooks, and if someone really wants the information from it they can implement the hook.